resources/qml/Preferences/GeneralPage.qml

```diff
@@ -659,6 +659,22 @@ UM.PreferencesPage
                     checked: boolCheck(UM.Preferences.getValue("info/send_slice_info"))
                     onCheckedChanged: UM.Preferences.setValue("info/send_slice_info", checked)
                 }
+
+                Button
+                {
+                    id: showMoreInfo
+                    anchors
+                    {
+                        top: sendDataCheckbox.bottom
+                        bottom: parent.bottom
+                    }
+
+                    text: catalog.i18nc("@action:button", "Show more information on anonymous data collection")
```

Comment:
Quite a lot of text for on a button. Maybe we can put just a little `?` there with this in the tooltip?

Reply:
Changed to "More Information" in 0d5b21932e1d87f696298e4b1d1d43ffac71ff11. There is a tooltip defined above.